### ISSUE TITLE
Adjusting complexity fails on hidden prims

### ISSUE THREAD

**Issue by aloysbaillet** [PixarAnimationStudios/USD#310]
### Description of Issue
When opening a stage in usdview, if prims are hidden and usdview was open with default complexity of 1, changing the complexity to more than 1 will cause this failure: `Error in 'pxrInternal_v0_8__pxrReserved__::HdStMesh::_GetRepr' at line 1388 in file usdImaging/pxr/imaging/lib/hdSt/mesh.cpp : '_InitRepr() should be called for repr refined on prim /xform/cube.'`
This is not a blocking issue as the workaround for now is to force a complexity value greater than 1 when starting usdview with the --complexity flag (or in maya to force it through hydra at stage opening time).

### Steps to Reproduce
1. save this usda file to /tmp/cube.usda:
```
#usda 1.0
def Xform "xform"{
    token visibility = "invisible"
    def Cube "cube"{
    }
}
```
2. Open this in usdview: `usdview /tmp/cube.usda`
3. In the `View->Adjust Complexity` menu choose `1.1` and type enter, you should see this output int he console:
```
Traceback (most recent call last):
  File "/tmp/rez_context_hBbzvU/PYTHONPATH/pxr/Usdviewq/stageView.py", line 2242, in paintGL
    self.drawSelHighlights)
  File "/tmp/rez_context_hBbzvU/PYTHONPATH/pxr/Usdviewq/stageView.py", line 1931, in renderSinglePass
    self._renderer.Render(pseudoRoot, self._renderParams)
pxr.Tf.ErrorException: 
	Error in 'pxrInternal_v0_8__pxrReserved__::HdStMesh::_GetRepr' at line 1388 in file /scratch/aloysb/dev/AL_USD/usdImaging/pxr/imaging/lib/hdSt/mesh.cpp : '_InitRepr() should be called for repr refined on prim /xform/cube.'
Traceback (most recent call last):
  File "/tmp/rez_context_hBbzvU/PYTHONPATH/pxr/Usdviewq/stageView.py", line 2108, in paintGL
    GL.glEnable(GL_FRAMEBUFFER_SRGB_EXT)
  File "errorchecker.pyx", line 53, in OpenGL_accelerate.errorchecker._ErrorChecker.glCheckError (src/errorchecker.c:1218)
OpenGL.error.GLError: GLError(
	err = 1282,
	description = 'invalid operation',
	baseOperation = glEnable,
	cArguments = (GL_FRAMEBUFFER_SRGB_EXT,)
)
Traceback (most recent call last):
  File "/tmp/rez_context_hBbzvU/PYTHONPATH/pxr/Usdviewq/stageView.py", line 2108, in paintGL
    GL.glEnable(GL_FRAMEBUFFER_SRGB_EXT)
  File "errorchecker.pyx", line 53, in OpenGL_accelerate.errorchecker._ErrorChecker.glCheckError (src/errorchecker.c:1218)
OpenGL.error.GLError: GLError(
	err = 1282,
	description = 'invalid operation',
	baseOperation = glEnable,
	cArguments = (GL_FRAMEBUFFER_SRGB_EXT,)
)
```

### System Information (OS, Hardware)
Linux, CentOS-6, NVidia driver version 378.13

### Package Versions
USD-0.8.1
OpenSubdiv-3.1.1

### Build Flags
OpenCL enabled

**Comment by jtran56** [comment]
Filed as internal issue #152898.

**Comment by sunyab** [comment]
We believe this should be fixed in the dev branch with commit 3ee4647f611125c38363f4234cc8af902099c207. 

If you get a chance to try this out, please let us know if that's not the case. In the meantime, marking as pending release for 0.8.2. Thanks!

**Comment by aloysbaillet** [comment]
That's great news thanks a lot! We'll try to cherry-pick this commit and we'll report back.

**Comment by sunyab** [comment]
Commit 3ee4647 has been merged into release 0.8.2, closing this out.
